app/controllers/partners_controller.rb

```diff
@@ -29,21 +31,29 @@ def index
   # GET /partners/1
   # GET /partners/1.json
   def show
-    # Period to show
-    @period = params[:period] || 'week'
-    @events = filter_events(@period, partner_or_place: @partner)
-    @opening_times = @partner.human_readable_opening_times
-
-    # Map
-    if @events&.length.positive?
-      @map = get_map_markers_from_events(@events)
```

Comment:
Just a note i conciously cut this because idk if it even works and it seems overkill